plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_multi_pt_mor.md

```diff
@@ -0,0 +1,60 @@
+## Create script
+Revision: 6f65998117a2d1228fc96d36053bd0d394499afe
+
+```scala
+test("Create MOR table with multiple partition fields with multiple types") {
+  withTempDir { tmp =>
+    val tableName = "hudi_multi_pt_mor"
```

Comment:
Mark the table version in the table and the zip name as well?

Reply:
Done